app/src/main/java/com/fstyle/androidtrainning/screen/tabhome/TabHomePresenter.java

```diff
@@ -1,15 +1,30 @@
 package com.fstyle.androidtrainning.screen.tabhome;
 
+import android.util.Log;
+import com.fstyle.androidtrainning.data.model.Movie;
+import com.fstyle.androidtrainning.data.service.config.MoviesApi;
+import com.fstyle.androidtrainning.data.service.response.GetListMoviesResponse;
+import com.fstyle.androidtrainning.utils.Constant;
+import java.util.List;
```

Comment:
ERROR: Unused import: java.util.List. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports